src/common.rs

```diff
@@ -0,0 +1,19 @@
+use console::{measure_text_width, Term};
+
+/// Returns a tuple with the height of the output and the number of lines that overflow the terminal width
+pub fn get_height(term: &Term, output: String) -> (usize, usize) {
+    let max_width = term.size().1 as usize;
+    let height = output.lines().count() - 1;
+    // calculate potential overflow of lines overflowing terminal width
+    let overflow = output
+        .lines()
+        .map(|l| {
```

Comment:
looks good to me. I feel like maybe with "fold" or "reduce" you could make it a tiny bit more concise but honestly I'd probably just do it the way you did